BusBoard.ConsoleApp/ApiHelper.cs

```diff
@@ -13,5 +14,30 @@ public static IEnumerable<T> ApiGet<T>(string baseUrl, string resource)
                 .Execute<List<T>>(new RestRequest(resource, DataFormat.Json))
                 .Data;
         }
+
+        public static PostcodeEntry GetPostcodeEntry(string postcode)
+        {
+            return ApiGet<PostcodeEntry>("https://api.postcodes.io/", $"postcodes/{postcode}").First();
+        }
+
+        public static IEnumerable<StopPointEntry.StopPoint> GetNearestStopPoints(PostcodeEntry postcodeEntry)
+        {
+            var resource =
+                $"StopPoint?stopTypes=NaptanPublicBusCoachTram&radius=400&useStopPointHierarchy=false&modes=bus&categories=none&returnLines=false&lat={postcodeEntry.result.latitude}&lon={postcodeEntry.result.longitude}";
+
+            return ApiGet<StopPointEntry>("https://api.tfl.gov.uk/", resource)
+                .First()
```

Comment:
Can this handle no stop points being returned?

Reply:
Yes it can, but should I add a conditional access?

Reply:
So what currently happens if the api returns no stop points? 🙂

Reply:
It returns an empty list and the foreach loop after it skips past.

Reply:
I see, that sounds fine 🙂